pkgs/development/interpreters/python/hooks/python-remove-tests-dir-hook.sh

```diff
@@ -0,0 +1,15 @@
+# Clean up top-level tests directory in site-package installation.
+echo "Sourcing python-remove-tests-dir-hook"
+
+pythonRemoveTestsDir() {
+    echo "Executing pythonRemoveTestsDir"
+
+    rm -rf $out/@pythonSitePackages@/tests
+
+    echo "Finished executing pythonRemoveTestsDir"
+}
+
+if [ -z "${dontRemoveTestsDir-}" ]; then
```

Comment:
`dontUsePythonRemoveTestsDir` not very short but consistent

Reply:
I'll align `poetry2nix` with this naming too.

Reply:
I think i renamed it, and forgot to update this. Thanks!

Reply:
although, i forgot about the `Use` portion anyway :)